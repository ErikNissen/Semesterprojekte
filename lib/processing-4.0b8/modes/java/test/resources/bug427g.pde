version https://git-lfs.github.com/spec/v1
oid sha256:ce0afdd6e91c32b7896fdbd13faf71641117b1e4781829891b207e30576435df
size 176
