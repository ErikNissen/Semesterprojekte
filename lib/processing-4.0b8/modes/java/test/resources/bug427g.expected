version https://git-lfs.github.com/spec/v1
oid sha256:6532643a02945ff8b7abf1451e8e65d7df9aab9eba61e1d5e6363156424d30d7
size 824
